libparsec/crates/types/src/pki/mod.rs

```diff
@@ -222,19 +222,6 @@ impl LocalPendingEnrollment {
             exc: e.to_string(),
```

Comment:
I guess the other methods `load_*` and `remove_*` are also scheduled to be moved into `platform_device_loader` in some future PRs ?

Reply:
Yes ;)